manifests/install.pp

```diff
@@ -192,7 +192,7 @@
   # Render the Shibboleth configuration. These are run time and not used
   # during the build process. It should restart Jetty, though.
   ['ldap.properties', 'idp.properties', 'authn/general-authn.xml', 'logback.xml',
-    'c14n/subject-c14n.xml', 'saml-nameid.xml',
+    'c14n/subject-c14n.xml', 'saml-nameid.xml', 'access-control.xml',
```

Comment:
It does now. It was previously relying on the inheritance from the shibboleth_idp class.